src/languages/elixir.js

```diff
@@ -10,111 +10,142 @@ Website: https://elixir-lang.org
 export default function(hljs) {
   const ELIXIR_IDENT_RE = '[a-zA-Z_][a-zA-Z0-9_.]*(!|\\?)?';
   const ELIXIR_METHOD_RE = '[a-zA-Z_]\\w*[!?=]?|[-+~]@|<<|>>|=~|===?|<=>|[<>]=?|\\*\\*|[-/+%^&*~`|]|\\[\\]=?';
-  const ELIXIR_KEYWORDS = {
+  const KEYWORDS = [
+    "alias",
+    "alias",
+    "and",
+    "begin",
+    "break",
+    "case",
+    "cond",
+    "defined",
+    "do",
+    "end",
+    "ensure",
+    "false",
+    "fn",
+    "for",
+    "import",
+    "in",
+    "include",
+    "module",
+    "next",
+    "nil",
+    "not",
+    "or",
+    "quote",
+    "redo",
+    "require",
+    "retry",
+    "return",
+    "self",
+    "then",
+    "true",
+    "unless",
+    "until",
+    "use",
+    "when",
+    "while",
+    "with|0"
+  ];
```

Comment:
I know that this PR merely reformats the list, but it caught my attention because there are a lot of keywords in it that I have never seen in Elixir. Also let my starts by saying that I am not sure if there is a formal definition of a "keyword". I'm using that term rather intuitively.


As far as I know, those are not Elixir keywords at all:
- `begin`
- `break` (`Inspect.Algebra.break/1` is a function in the standard lib, but it has nothing to do with breaking out of loops like it might do in other languages)
- `defined`
- `ensure`
- `include`
- `module`
- `next` (`OptionParser.next/2` is a function in the standard lib, it has nothing to do with skipping to another iteration in a loop like it might do in other languages)
- `redo`
- `retry`
- `return`
- `until`
- `while`

I'm unsure about:
- `self`. It exists, but IMO it is a normal macro, not a keyword. I wouldn't expect it to be colored differently than other normal macros (like `rem`, `round`, `trunc` etc).
- `then`. It's a new macro added in Elixir 1.12 together with `tap`, I don't think those are keywords.
- `with|0`. I'm not sure what the pipe and zero mean here. `with` on its own is definitely a keyword.

Missing IMO:
- if `case`, `cond`, and `unless` qualify as keywords, `if` and `else` should too.
- if `quote` qualifies as a keyword, `unquote` and `unquote_splicing` should too.
- `receive`/`after` https://hexdocs.pm/elixir/Kernel.SpecialForms.html#receive/1
- `try`/`rescue`/`catch`/`after` https://hexdocs.pm/elixir/Kernel.SpecialForms.html#try/1
- `raise` https://hexdocs.pm/elixir/Kernel.html#raise/1
- `reraise` https://hexdocs.pm/elixir/Kernel.html#reraise/2

Reply:
`alias` is also listed twice in the list.